lib/scholar/manifold/trimap.ex

```diff
@@ -0,0 +1,478 @@
+defmodule Scholar.Manifold.Trimap do
+  @moduledoc """
+  TriMap: Large-scale Dimensionality Reduction Using Triplets.
```

Comment:
It you can add a bit more words, especially about how it compares with other techniques in Scholar (such as kNN), both in terms of performance and results, it would be excellent. :)